VnE/Source/Extensions/ChemPack/voronoi.py

```
@@ -0,0 +1,132 @@
+        if sym_codes:
+            cell = [atoms[0].cif_cell_a,
+                    atoms[0].cif_cell_b,
+                    atoms[0].cif_cell_c,
+                    atoms[0].cif_cell_al,
+                    atoms[0].cif_cell_be,
+                    atoms[0].cif_cell_ga]
+        else:
+            ...
+
+        symms = [x[1] for x in sym_codes]
+        data = [tuple([x.atom_type, *list(x.cif_frac_coords)]) for x in atoms]
```

Ruff (0.14.14)

[convention] Unnecessary list literal passed to `tuple()` (rewrite as a tuple literal)

Rewrite as a tuple literal

(C409)